python/hail/docs/getting_started_developing.rst

```diff
@@ -0,0 +1,66 @@
+.. _sec-getting_started_developing:
+
+==========================
+Getting Started Developing
+==========================
+
+You'll need:
+
+- The `Java 8 JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_.
+- `Spark 2.0.2 <http://spark.apache.org/downloads.html>`_
+- `Anaconda <https://www.continuum.io/downloads>`_.
+
+-------------------
+Building a Hail JAR
+-------------------
+
+See :ref:`building-hail-from-source`.
+
+-----------------
+Building the Docs
+-----------------
+
+Hail uses conda environments to manage python dependencies. First, create a
+conda environment for hail:
+
+.. code-block:: bash
+
+    conda env create hail -f ./python/hail/environment.yml
+
+And run the ``createDocs`` gradle task in the environment:
+
+.. code-block:: bash
+
+    (source activate hail && gradle createDocs) ; source deactivate hail
```

Comment:
I've never used conda environments.  Is this how you're working?  What are the pros and cons of working this way?  We have to install the decorator dependency on the cloud (and soon autopep8 for history).  We don't mention that anywhere in the user getting started.  Is that just luckily default with conda?  So we be suggesting using conda environments for users and developers all?

Personally, I just want to install the dependencies from the environment file in my default conda environment.  Is that possible?  That might be a better workflow.  I didn't immediately see how to do that.